api/service.go

```diff
@@ -97,15 +98,15 @@ func (s *Service) RegisterValidator(ctx context.Context, receivedAt time.Time, p
 			}
 			out, err := c.RegisterValidator(clientCtx, req)
 			if err != nil {
-				errChan <- toErrorResp(http.StatusInternalServerError, err.Error(), id, "relay returned error", clientIP)
+				errChan <- toErrorResp(http.StatusInternalServerError, err.Error(), "", id, "relay returned error", clientIP)
 				return
 			}
 			if out == nil {
-				errChan <- toErrorResp(http.StatusInternalServerError, "failed to register", id, "empty response from relay", clientIP)
+				errChan <- toErrorResp(http.StatusInternalServerError, "", "failed to register", id, "empty response from relay", clientIP)
```

Comment:
do not need to respond this to follow same as relay
```suggestion
				errChan <- toErrorResp(http.StatusInternalServerError, "", "", id, "empty response from relay", clientIP)
```

Reply:
Fixed